c/ckb.mol

```diff
@@ -0,0 +1,363 @@
+// Copy from ckb-types#656cb6e5
```

Comment:
Only half of these structs are required.

When I finished the `import` statement grammar, we can split there structs into several schema files, then this crate could only include the schema file about blockchain.